bitcoin/src/blockdata/block.rs

```diff
@@ -305,6 +305,13 @@ impl Block {
     pub fn coinbase(&self) -> Option<&Transaction> { self.txdata.first() }
 
     /// Returns the block height, as encoded in the coinbase transaction according to BIP34.
+    /// 
+    /// # Errors
+    /// 
+    /// Block version is less than 2 [`Bip34Error::Unsupported`], 
+    /// Coinbase Tx or the first input in Coinbase Tx is absent [`Bip34Error::NotPresent`], 
+    /// First instruction of first input is not push instruction [`Bip34Error::UnexpectedPush`], 
+    /// Block height is negative [`Bip34Error::NegativeHeight`].
```

Comment:
I noticed overflow (too large push) is not listed here. If you didn't forget about it we may have a bug.